src/types/Todos.ts

```diff
@@ -0,0 +1,11 @@
+import User from './User';
+
+export interface Todos {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user?: User | null
+}
+
+export type ArrayOfTodos = Todos[];
```

Comment:
This type is not needed. `Todos[]` is a known syntax for every typescript dev. Using a custom type ArrayOfTodos is redundant.
```suggestion
```